### PR TITLE
create past activity log

### PR DESCRIPTION
**Pull request by devonwigle**
## Branch title

- feature/past activity card functionality dw

## Changes made

-  add section for past activity log
- add innerHTML for past activity log

## Anything Refactored

- move hidden and visible to bottom of CSS

## Anything needed

-  Fix the color of the past log
-  add new activity button

## Contributors

* Stevie Ambroise
* Alex Fritz
* Devon Wigle

### CODE REVIEW

File: src/Activity.js [hunk 1]
```diff
@@ -1,5 +1,5 @@
 class Activity {
-  constructor(category, description, minutes, seconds) {
+  constructor(category, description, minutes, seconds, color) {
```
Comment (StevieAmb) [inline src/Activity.js:2]: This was a great way to work to get the colors on the page!

File: index.html [hunk 2]
```diff
@@ -83,14 +83,6 @@ <h2 class="activity-title" id="pastActivities">Past Activities</h2>
          <p class="no-logged-activities">Complete the form to the left to get started!</p>
         </section>
         <section class="past-activities-box hidden">
-          <section class="past-activities">
```
Comment (alexmfritz) [inline index.html:86]: Bye bye HTML

File: styles.css [hunk 3]
```diff
@@ -303,14 +303,6 @@ aside {
   width: 40px;
 }
 
-.hidden {
```
Comment (alexmfritz) [inline styles.css:306]: Who done this?!

File: src/main.js [hunk 4]
```diff
@@ -46,6 +48,11 @@ startActivityButton.addEventListener('click', startActivity);
 studyButton.addEventListener('click', highlightStudy);
 meditateButton.addEventListener('click', highlightMeditate);
 exerciseButton.addEventListener('click', highlightExercise);
+logActivityButton.addEventListener('click', function(event) {
```
Comment (StevieAmb) [inline src/main.js:51]: This is a great use of the anonymous function!

**Review by StevieAmb — APPROVED** [review]

**Pull request merged**